### ISSUE TITLE
No API key provided

### ISSUE THREAD

**Issue by rock5791** [ron4mac/joomla_mod_stripe_donate#1]
1.There is no issue. I am new and this is my first module I am working with. I am getting error below while I have inserted the keys that I copied from my stripe dashboad. I wonder if there is something else that needs to be done in addition to this. Super new, so sorry for being vague. 

No API key provided. (HINT: set your API key using "Stripe::setApiKey()". You can generate API keys from the Stripe web interface. See https://stripe.com/api for details, or email support@stripe.com if you have any questions.
2. I tried to play around to see if I can change/adjust the way the form looks like but in vain. Here again, not your fault, just me being a newby

**Comment by ron4mac** [comment]
Make sure you are using a version that you downloaded no longer ago than the last update made to the module (I have not made version number changes for anything yet).

**Comment by ron4mac** [comment]
Also, you can look at the file 'FCPPAY.LOG' in your joomla root for maybe some hint. It should show what the stripe key was set to.

**Comment by rock5791** [comment]
Hello Ron4mac, thanks for your response. I spent some times with stripe and we fixed the issue. the problem was on their site. I will work on changing what I can. I wanted to see if there was a possibility of adding two rows to say  employer information and occupation since it is a campaign site(aliciabrooksforjudge.com ), one of those requirements.
2. Can make a donation to some of your project ect... thank you for this module.
